discord/commands/commands.py

```diff
@@ -857,6 +857,7 @@ def __init__(
         )
         self.name = name
         self.description = description
+        self.input_type = SlashCommandOptionType.sub_command_group
```

Comment:
I believe this fix is not correct as the problem is super calling the init of `ApplicationCommand` instead of `Option`. The correct fix would be to fix the super call at line 853. One possible fix for this would be to replace the init call with this to correctly call the init of `Option`.
```py
Option.__init__(
    self,
    SlashCommandOptionType.sub_command_group,
    name=name,
    description=description,
)
```
This would correctly set input_type for the class.

There are other solutions as well as can be seen in [this stackoverflow answer](https://stackoverflow.com/a/39415209/6046713).

Reply:
This is just a temporary fix, we were discussing something like this in the `#discussion` channel in discord.gg/pycord